test/scripts/setup-konvoy.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+# NOTE: used by teamcity
+
+KONVOY_VERSION="v1.4.0"
```

Comment:
Lets use `v1.4.1` as it contains a few bug fixes.

Reply:
```suggestion
KONVOY_VERSION="v1.4.1"
```